igmspec/defs.py

```diff
@@ -40,6 +40,7 @@ def instruments():
         # HST
         'ACS': dict(gratings=['PR200L']),
         'WFC3': dict(gratings=['G280']),
+        'COS': dict(gratings=['G130M', 'G160M', 'G130M/G160M']),
         # VLT
```

Comment:
Are you planning to ingest the HIRES COS-Halos data too? I don't think it looks like it here...